be/src/common/config.h

```diff
@@ -669,6 +669,13 @@ CONF_Int64(max_segment_file_size, "1073741824");
 // hdfsPreadFully() are always enabled for object storage.
 CONF_Bool(use_hdfs_pread, "true");
 
+// rewrite partial semgent or not
+// if true, partial segment will be rewrite into new segment file firset and append other column data
```

Comment:
firset -> first？

Reply:
> firset -> first？

done, fix in https://github.com/StarRocks/starrocks/pull/2969/commits/3b64bfc26e026712b1192db93e41235b2e1a9c53